packages/git/src/browser/git-widget.tsx

```diff
@@ -311,6 +313,12 @@ export class GitWidget extends ReactWidget implements StatefulWidget {
                 {this.renderCommandBar(repository)}
             </div>
             <div className='changesOuterContainer' id={this.scrollContainer}>
+                {
+                    this.incomplete ?
+                        <div className='theia-git-message theia-git-message-warning'>
+                            There are too many active changes, only a subset is shown.</div>
```

Comment:
Why do not we mimic VS Code and show a warning instead:
https://github.com/Microsoft/vscode/blob/014737f9a07e66deb9e8efdc00d6915c34ed4a44/extensions/git/src/repository.ts#L1063-L1067

If no, can we make the warning always visible even if the user scrolls down?
![screencast 2018-08-07 10-29-20](https://user-images.githubusercontent.com/1405703/43764412-b19e7a56-9a2d-11e8-8369-76d246c64aa4.gif)

Or at least, please make sure the scroll-bar does not overlay the warning div. It looks ugly.
![screencast 2018-08-07 10-41-10](https://user-images.githubusercontent.com/1405703/43764855-da0ca548-9a2e-11e8-9399-d5dcc7bbccff.gif)